tests/zfs-tests/tests/functional/quota/quota.kshlib

```diff
@@ -57,7 +57,7 @@ function fill_quota
 	$FILE_WRITE -o create -f $MNTPT/$TESTFILE1 -b $BLOCK_SIZE \
 		 -c $write_size -d 0
 	zret=$?
-        [[ $zret -ne EDQUOT ]] && \
+        [[ $zret -ne $EDQUOT ]] && \
```

Comment:
Exactly which quota tests does this fix and can we enable a few more which are currently disabled in the runfile?  If so let's do it.